Gemfile.lock

```diff
@@ -190,11 +190,11 @@ GEM
       responders
       warden (~> 1.2.3)
     digest (3.1.0)
-    erubi (1.10.0)
+    erubi (1.11.0)
```

Comment:
### [erubi](https://github.com/jeremyevans/erubi)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.10.0 → 1.11.0

[_[change-log](https://github.com/jeremyevans/erubi/blob/master/CHANGELOG), [source-code](https://github.com/jeremyevans/erubi), [gem-diff](https://my.diffend.io/gems/erubi/1.10.0/1.11.0)_]





<details>
<summary>Commits</summary>

A change of **18** commits. See the full changes on [the compare page](https://github.com/jeremyevans/erubi/compare/1.10.0...1.11.0).

These are the first 10 commits:
- (d18a88f) [Use GitHub Actions CI for supported Ruby versions](https://github.com/jeremyevans/erubi/commit/d18a88fd0a89ad8468bf889cb339484f7551e992)
- (c8c768a) [RANGE_ALL is not in use since 4dc81c210664bfa244c6015bb3aa03…](https://github.com/jeremyevans/erubi/commit/c8c768abe59434f940ed628d93c7fc9c32562bea)
- (109499f) [Test on ruby 3.0](https://github.com/jeremyevans/erubi/commit/109499f2f647e1ef2765c872c855b751e0a7e8ae)
- (32f91ea) [Start testing on truffleruby, and simplify ci.yml](https://github.com/jeremyevans/erubi/commit/32f91eaacbbf43158f7335add04986b938b57507)
- (b949629) [Bump copyright year](https://github.com/jeremyevans/erubi/commit/b949629c68994e3fd1f46837d5d86401c7390571)
- (7245719) [Stop using Travis](https://github.com/jeremyevans/erubi/commit/72457190c4edbd8d330d72b1e801f08925f83048)
- (595dfaf) [Extract default regexp to Constant](https://github.com/jeremyevans/erubi/commit/595dfaff86e3dd9dff09cbdfc8bc043bfcd64c6f)
- (ae72c9d) [Try Ruby 1.9.3, 2.0, and JRuby 9.3 on GitHub Actions](https://github.com/jeremyevans/erubi/commit/ae72c9df5d98f9c8b03ec8ed1e3ca6a994bafc3e)
- (a9f8a0d) [Run specs in verbose mode on Ruby 3+](https://github.com/jeremyevans/erubi/commit/a9f8a0dc91c874ba41cbfa77e059bbb6253f7fa0)
- (7770b01) [Test Ruby 3.1 in CI](https://github.com/jeremyevans/erubi/commit/7770b01cfa2b60016f8f6d6b714ab6accf7bccfb)


</details>